File: Modul 05/05/PRAK405-2410817310020-MRizkiDinar.c
#include <stdio.h>

int main() {
    int val, mult, row = 0, SUM = 0;
    scanf("%d %d", &val, &mult);
    
    for (int i = 2; i < val + 2; i++) {
        for (int j = i - 1; j >= 1; j--) {
            row += (j * mult);
            printf("(%d * %d)", j, mult);
            if (j == 1) {
                printf(" = ");
            } else {
                printf(" + ");
            }
        }
        printf("%d\n", row);
        SUM += row;
        row = 0;
    }
    printf("%d\n", SUM);
    
    return 0;
}

File: Modul 04/05/PRAK405-2410817310020-MRizkiDinar.py
val, mult = map(int, input().split())
row = 0
SUM = 0

for i in range(2, val+2):
    for j in reversed(range(1 ,i)):
        
        row+= (j * mult)
        print(f"({j} * {mult})", end = "")
        if j == 1:
            print(end = " = ")
        else:
            print(end = " + ")
        
    print(row)
    SUM += row
    row = 0
print(SUM)